R/validate_inputs.R

```diff
@@ -68,3 +68,31 @@ assert_area_id_exists <- function(json) {
   }
   invisible(TRUE)
 }
+
+#' Validate population file.
+#'
+#' Check that population file can be read.
+#'
+#' @param shape Path to input population file.
+#'
+#' @return An error if invalid.
+#' @keywords internal
+do_validate_population <- function(population) {
+  population <- read_csv(population, header = TRUE)
+  ## Perhaps this kind of assertion should just be checked by json schema?
+  assert_column_names(
+    colnames(population),
+    c("iso3", "area_id", "time", "sex", "age_group_id", "source", "population"))
+  NULL
```

Comment:
I don't think there is any data we want to return here. This data is only used for the model run (I don't know of any plot which requires it yet) but we can always return something later down the line.